src/gui/CloneDialog.cpp

```diff
@@ -57,6 +58,7 @@ void CloneDialog::cloneEntry()
     Entry* entry = m_entry->clone(flags);
     entry->setGroup(m_entry->group());
 
+    m_parent->entryView()->setCurrentEntry(entry);
```

Comment:
Both of these lines really need to go away and be replaced by either a finish signal or the parent listens for the standard finish signal and performs post clone actions.

Reply:
Thanks for the review.  Do you mean something like this?  c2dc24c4

Reply:
Yes perfect